ecommerce/pricing/lib/pricing/happy_hour.rb

```diff
@@ -0,0 +1,28 @@
+module Pricing
+  class HappyHour
+    include AggregateRoot
+
+    AlreadyCreated = Class.new(StandardError)
+
+    def initialize(id)
+      @id = id
+    end
+
+    def create(**kwargs)
+      # This is an implicit validation based on the @id only.
```

Comment:
instead of code comments - we prefer to create tickets if it's a technical debt

Reply:
Done - https://github.com/RailsEventStore/ecommerce/issues/143